cmd/clusterctl/client/cluster/mover_test.go

```diff
@@ -1184,7 +1221,15 @@ func Test_objectMover_move(t *testing.T) {
 				fromProxy: graph.proxy,
 			}
 
-			err := mover.move(graph, toProxy)
+			// choose to include/exclude mutator randomly
+			includeMutator := randSource.Int63()%2 == 0
```

Comment:
I'm personally -1 to have random behavior in tests because this will make it harder to chase errors or flakes.
I suggest instead creating a new unit test where we always use mutators.

Reply:
+1 Good call. Sorry, I missed this in my review.

Reply:
i cloned the existing test case and passed mutators to move unconditionally. thanks for the suggestion!